tic_tac_toe/game.py

```diff
@@ -24,7 +28,14 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
+
+    valid_positions = (
+        (0, 0), (0, 1), (0, 2),
+        (1, 0), (1, 1), (1, 2),
+        (2, 0), (2, 1), (2, 2),
+    )
+
+    return True if position in valid_positions else False
 
```

Comment:
It's visually intuitive, but I really like the programmatic solution this team provided:
https://github.com/rmotr-group-projects/pyp-w1-gw-tic-tac-toe/pull/50